content/exchange/artifacts/Server.Alerts.Mail.yaml

```diff
@@ -0,0 +1,551 @@
+       LET Link(URL, Name) = format(format=if(condition=HTML,
+                                              then='<a href="%[1]s">%[2]s</href>',
+                                              else='%[2]s (%[1]s)'),
+                                    args=(URL, Name))
+
+       LET OrgInfo <= org()
+
+       // Return unique values from a list:
+       LET Unique(Items) = items(item=to_dict(item={
+           SELECT _value AS _key
+           FROM foreach(row=Items)
+         }))._key
+
+       LET CreatorAddrs = if(condition=NotifyExecutor
+                              AND Creator =~ '[^@]+@.+',
+                             then=(Creator, ),
+                             else=[])
+
+       // Main information dict used in the HTML e-mail:
+       LET TableRowsDict = dict(`Organisation`=if(
+                                  condition=OrgInfo.id != 'root',
+                                  then=OrgInfo.name,
+                                  else=NULL),
+                                `Client`=Link(URL=link_to(
+                                                client_id=
+                                                  ClientId,
+                                                raw=true),
+                                              Name=FQDN),
+                                `New client`=NewClient) +
+           ClientMetadata + dict(
+           `Flow`=Link(URL=link_to(client_id=ClientId,
+                                   flow_id=FlowId,
+                                   tab='results',
+                                   raw=true),
+                       Name=FlowId),
+           `Collection created`=TimestampString(Timestamp=FlowCreated),
+           `Collection started`=TimestampString(Timestamp=FlowStarted),
+           `Collection finished`=TimestampString(Timestamp=FlowFinished),
+           `Duration`=Duration,
+           `Creator`=Creator,
+           `Requested`=BulletList(Items=Requested),
+           `Arguments`=ArgumentTable,
+           `With results`=BulletList(Items=WithResults),
+           `Error`=if(condition=Status, then=Status),
+           `Urgent`=Urgent,
+           `Hunt`=if(
+             condition=NotifyHunts
+              AND OurHunt.HuntId,
+             then=Link(URL=link_to(hunt_id=OurHunt[0].HuntId, raw=true),
+                       Name=OurHunt[0].HuntDesc || OurHunt[0].HuntId)),
+           `Collected rows`=CollectedRows,
+           `Uploaded files`=UploadedFiles,
+           `Uploaded bytes`=if(condition=UploadedBytes,
+                               then=humanize(bytes=UploadedBytes),
+                               else=0))
+
+       LET Title = format(format='Client collection %v', args=ResultWord)
+
+       LET PlainText = format(format='%v\n\n%v\n\n%v',
+                              args=(Title, Summary, Table(Values=TableRowsDict)))
+
+       // Add some CSS to make the table look at least a bit nice:
+       LET Body = if(
```

Comment:
Might be easier to use the new template() function

https://docs.velociraptor.app/vql_reference/other/template/

Reply:
After further thought I realized that the template feature is only available in the head release and so this makes the artifact not work on older servers. This might be OK for you though? 

We should hopefully have an RC out very soon.